.tekton/cert-manager-istio-csr-1-16-pull-request.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "release-1.15" &&
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "release-1.16" &&
```

Comment:
We need to update application, component and image names too. Or we can modify the config created by konflux-bot once after creating new application and components in konflux for 1.16 release.

Reply:
Per https://gitlab.cee.redhat.com/releng/konflux-release-data/-/merge_requests/5788, the konflux objects were applied. We can let the bot update the tekton configs as discussed (instead of manual changes, dropped the commit which manually updated `.tekton/`)

Reply:
Ack, please make sure to customization the configs and also to delete the `1-15` configs in `tekton` dir.